views/select_item.erb

```diff
@@ -0,0 +1,21 @@
+<div>
+    <h3>Happy to have you here:</h3>
+    <h1><%= @player_1_name %>!</h1>
+</div><br>
+
+<div>
+    <label for="item_choice_selector">Please make your choice :</label>
+
+    <form action="/item_choice" method="post">
```

Comment:
nice way to show your options with a drop down.  can you show me how this works in the browser if we have time